src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  cursor: pointer;
+  margin-right: auto;
+  margin-left: 50px;
+}
+
+header {
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  align-items: center;
+}
+
+nav {
+  margin-right: 50px;
+}
+
+.nav__list {
+  list-style: none;
+}
+
+li,
+a {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-decoration: none;
+  color: #000;
+}
+
+.is-active {
+  margin-left: 0;
+  color: #00acdc;
+  position: relative;
+}
+
+.nav__list li {
+  display: inline-block;
+  padding-top: 23px;
+  padding-bottom: 23px;
+  text-transform: uppercase;
+}
+
+a {
+  padding-top: 23px;
+  padding-bottom: 23px;
+  margin-left: 16px;
+}
+
+.nav__list li a:hover {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  right: 0;
+  top: 56px;
+  height: 4px;
+  width: 37px;
```

Comment:
If you replace it with `left: 0` it will work with any width,

Reply:
I agree with Paul. the reason this element is 37px is that the width of the first link `APPLE` is 37 pixels. but what if I change the font? I have to remember to come and change this as well. what if I make another element active, let's say `SAMSUNG`, it has a different width. make it be 100% of the link. you can achieve this using width or as Paul suggested, use left 0 and right 0

look what happens if I make the `SAMSUNG` active
![Screenshot 2022-03-21 at 18 57 11](https://user-images.githubusercontent.com/79282749/159326516-82189771-f3fa-435a-8ce3-fdf5db24dee3.jpg)

the design is broken